edm4eic.yaml

```
@@ -244,6 +244,22 @@ datatypes:
   ## ==========================================================================
   ## Calorimetry
   ## ==========================================================================
+  edm4eic::SimCalorimeterPulse:
+    Description: "Simulated calorimeter pulse prior to digitization, implements SimPulse."
+    Author: "D. Anderson, S. Joosten"
+    Members:
+      - uint64_t            cellID          // ID of the readout cell for this pulse.
+      - float               integral        // Total pulse integral in relevant units.
+      - edm4hep::Vector3f   position        // Position of the hit in world coordinates [mm].
+      - float               time            // Start time for the pulse in [ns].
+      - float               interval        // Time interval between amplitude values [ns].
+    VectorMembers:
+      - float               amplitude       // Pulse amplitude in in relevant units, sum of amplitude values equals integral
+    OneToManyRelations:
+      - edm4hep::SimCalorimeterHit hits     // SimCalorimeterHits used to create this pulse
+      - edm4eic::SimCalorimeterPulse pulses // SimCalorimeterPulses used to create this pulse
+      - edm4hep::MCParticle particle        // MCParticle that caused the pulse
```

```suggestion
      - edm4hep::MCParticle particles       // MCParticle that caused the pulse
```
All one to many relations are plural, it looks like.